doc/lua_api.txt

```diff
@@ -68,9 +68,12 @@ The game directory can contain the following files:
       e.g. `allowed_mapgens = v5,v6,flat`
       Mapgens not in this list are removed from the list of mapgens for
       the game.
+      If not specified, all mapgens are allowed.
     * `disallowed_mapgens = <comma-separated mapgens>`
       e.g. `disallowed_mapgens = v5,v6,flat`
       These mapgens are removed from the list of mapgens for the game.
+      When both `allowed_mapgens` and `disallowed_mapgens` are both
+      specified, `allowed` is applied before `disallowed'.
```

Comment:
First "both" is redundant. Code tags should contain code or complete setting names.

Suggestion: `allowed_mapgens` is applied first when used in combination.

Reply:
@SmallJoker, Thanks for that feedback. I update dropped a new commit. How do you feel about this one?